recipe/meta.yaml

```diff
@@ -28,14 +27,14 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6
+    - python
 
 test:
   imports:
     - pycodestyle
   requires:
     - pip
-    - python <3.10
+    - python
```

Comment:
python can be removed from test/requires. It's already in `run`

Reply:
Thanks Serhii!